pom.xml

```diff
@@ -234,6 +234,7 @@
 
     <!-- Solve conflict across dependencies -->
     <kotlin.stdlib.version>1.9.24</kotlin.stdlib.version>
+    <jetbrains.annotations.version>13.0</jetbrains.annotations.version>
```

Comment:
Can you try using the latest version (24.1.0) of this library and see if it works